src/components/Header/SearchBar.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./Header.scss";
+export default function SearchBar() {
+  return (
+    <div className="search-bar">
+      <label>
+        <input
+          type="text"
+          placeholder="Search for products"
+          class="desktop-search-bar"
+          value={""}
+        />
+      </label>
+      <span class="search-icon">
+        <svg
```

Comment:
Lets create a folder `src/components/icons`
And put all inline svg's as a React component there.
So we may move the SVG into a component `src/components/icons/Search.js`

**Bonus**: checkout this tool [Transform tool , create optimized SVG JSX](https://transform.tools/)